automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -293,9 +293,10 @@ def product_upgrade(
     if product == 'capsule':
         if not os.environ.get('CAPSULE_HOSTNAME'):
             if not cap_image and not os.environ.get('RHEV_CAPSULE'):
-                print('Please provide either Capsule RHEVM template name or '
-                      'Capsule HostName to perform upgrade!')
-                sys.exit(1)
+                cap_image = os.environ.get('CAP_IMAGE')
+                cap_host = os.environ.get('CAP_HOST')
```

Comment:
these variables `CAP_IMAGE` and `CAP_HOST` are defined in jenkins under `RHEV_Config`